### DISCUSSION TITLE
How to deploy docker compose from CLI?

### DISCUSSION THREAD

**Discussion by alexisLefebvre** [caprover/caprover#2137] [Q&A] [answered]
Here it is explained how to deploy a docker compose stack: https://caprover.com/docs/docker-compose.html#how-to-run-docker-compose-on-caprover

Is it possible to deploy a file like this through the CLI instead of the GUI? The goal is to deploy Docker image on CapRover through GitLab CI.

**Comment by githubsaturn** [comment, marked as answer]
No unfortunately. Docker compose apps can only be deployed on the GUI.
